tests/fake_backends/test_fake_deneb.py

```diff
@@ -159,7 +171,7 @@ def test_execute_works_but_backend_run_doesnt_with_unsupported_gates():
     qc_list[3].z(4)
 
     for qc in qc_list:
-        execute(qc, backend, shots=1000)
+        backend.run(transpile_to_IQM(qc, backend=backend), shots=1000)
```

Comment:
The name of this test `test_execute_works_but_backend_run_doesnt_with_unsupported_gates` is now outdated, could be renamed to something like `test_backend_run_doesnt_work_with_unsupported_gates`.

Reply:
Renamed it to `test_transpiling_works_but_backend_run_doesnt_with_unsupported_gates`

Reply:
(looks like it's not actually renamed yet)